solutions/src/main/java/ru/sberbank/school/task06/CountMap.java

```diff
@@ -0,0 +1,77 @@
+package ru.sberbank.school.task06;
+
+import java.util.Map;
+
+/**
+ * Интерфейс мапы, которая ведет подсчёт вхождений элемента в контейнер.
+ *
+ *
+ * Пример(без дженериков):
+ *
+ * CountMap map = new ... ();
+ *
+ * map.add("Some word");
+ *
+ * map.add("Some word");
+ *
+ *
```

Comment:
Empty line should be followed by &lt;p&gt; tag on the next line.

*<this is a auto generated comment from violation-comments-lib F7F8ASD8123FSDF>* *<a-57223586>*